src/util/beautifier-maps.ts

```diff
@@ -434,4 +434,59 @@ export const BEAUTIFIER_EVENT_MAP = {
     z: 'orderFilledAccumulatedQuantity',
     T: 'orderTradeTime',
   },
+
+  contractInfo: {
```

Comment:
If this value comes from `eventType` (or `e`), then it needs to be suffixed with `Event` (take a look at `forceOrderEvent` above, it's based on `e: 'forceOrder'`)